communication/slack-config/channels.yaml

```diff
@@ -441,9 +440,12 @@ channels:
   - name: sig-contribex-meetings
   - name: sig-contribex-comms
   - name: sig-contribex-elections
+  - name: sig-contribex-lwkd
```

Comment:
Can I please double check this is intentional?  Not sure this relates to sig-etcd?

Reply:
ooops, good catch.  Please wait for force push